lib/vagrant-service-manager/installer.rb

```diff
@@ -0,0 +1,106 @@
+module VagrantPlugins
+  module ServiceManager
+    BINARY_NAME = {
```

Comment:
The is used in docker or openshift binary handlers which is on similar level at `Installer`.